articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md

```diff
@@ -0,0 +1,53 @@
+
+Often, some organizations may not have a precise naming convention for workloads and applications. This means that network security teams have to spend significant time attaching different workloads to given applications.
+
+These activities are time-consuming, and the use of artificial intelligence technology can come in handy. AI can expedite these processes by learning and observing traffic patterns and suggesting security policies. This way, organizations save on time, resources, and effort. The organization can allocate these to other areas of technological advancement or development.
+
+### Data Centers
+
+Cyber attackers target [data centers](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/what-is-a-data-center.html) in an attempt to obtain critical data. Cybersecurity has improved over the years with advancements in technology. Being aware of this, cybercriminals regularly develop more advanced malware strains to ensure successful cyber-attacks on organizations&#39; networks.
+
+Organizations can deploy artificial intelligence technology in their data centers to enhance data security. To ensure data security, AI essentially learns normal network behavior. It then detects cyber threats with regard to the variations from the regular network behavior.
+
+The application of AI in a data center can help identify security loopholes and detect malware in data center systems. Most importantly, AI-based cybersecurity can analyze or screen outgoing and incoming data security threats.
+
+### Threat Hunting
+
+The asymmetric nature of cybersecurity is an inherent problem that seems to empower the acts of attackers. With this state, cyber-attackers need to only be successful once by exploiting just a single weakness while the defenders must be successful every time. Preventing threats requires more than just ensuring security data. It calls for combing threats on an entire [information technology stack](https://www.pcmag.com/encyclopedia/term/technology-stack).
+
+In this case, AI is essential to identify abnormalities, outliers, and patterns in all data without the need for applying fixed rules. Human analysts depend on this output for investigation and action. This is what is often referred to as threat hunting in security language. It involves narrowing down threats by collaborating security analytics and machine intelligence, and advanced human cognition.
+
+### Vulnerability Management
+
+Every year, there are many reported vulnerabilities to cybersecurity. It is extremely difficult to manage these unique vulnerabilities with traditional technology or human resources. Systems that use machine learning and artificial intelligence do not wait for online attackers to exploit existing vulnerabilities. Rather, these systems are proactive in finding potential vulnerabilities in information systems.
```

Comment:
How is vulnerability different from threat hunting? I feel we are emphasizing the point that AI solves the problem. But, as a reader, I don't understand how is it working? Divide the explanation into the following steps:
1. Problem statement: What are we trying to solve?
2. How is AI solving it(mention a few algorithms, techniques, models, if any)?
3. Advantages of AI over other systems
4. How AI integrates with the problem and solves it.